### PR TITLE
Document watchdog

### PR DESCRIPTION
**Pull request by albsod**
This describes general Linux watchdog usage for all platforms and adds a page for HMX about its M7 watchdog.

### CODE REVIEW

File: docs/system/hmx/watchdog.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Watchdog (HMX)
+tags:
+  - Watchdog
+  - HMX
```
Comment (RikardO-HM) [inline docs/system/hmx/watchdog.md:5]:
```suggestion
  - HMX
  - MX-4
  - CT
  - CT GL
  - MX-V
  - T30
  - T30 FR
  - C61
```
better to use all tags

File: docs/system/watchdog.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+title: Watchdog
+tags:
+  - Watchdog
+  - HMX
+  - MX-4
+  - MX-V
```
Comment (RikardO-HM) [inline docs/system/watchdog.md:7]:
```suggestion
  - HMX
  - MX-4
  - CT
  - CT GL
  - MX-V
  - T30
  - T30 FR
  - C61
```

File: docs/system/hmx/watchdog.md (hunk 1 above)
Comment (albsod) [reply docs/system/hmx/watchdog.md:5]: not here though

Comment (RikardO-HM) [reply docs/system/hmx/watchdog.md:5]: correct my bad

**Comment by albsod** [comment]
Thanks, I included the tags to the Linux watchdog page.

**Review by RikardO-HM — APPROVED** [review]

**Pull request merged**